application/base.py

```diff
@@ -24,7 +24,7 @@ def decorate(instance, data):
 class BaseSchema(Schema):
     """ Initialize Schemas with Schema(Model) """
 
-    def __init__(self, model, *args, decorate=False, create_objects=True, **kwargs):
+    def __init__(self, *args, model=None, decorate=False, create_objects=True, **kwargs):
```

Comment:
This being a required argument messed with Nested fields. So I made a keyword argument out of it. @Adrian-St note for you and me for the future. When building wrappers don't make arguments required and always provide default values.